packages/plugins/test/mocha/cli/propose.spec.ts

```diff
@@ -167,14 +171,14 @@ describe('Propose CLI command', () => {
     const scriptPath = 'contracts/test/script/Simple.s.sol'
     const isTestnet = false
     const targetContract = 'Simple1'
-    const context = makeMockSphinxContext([
+    const { context, prompt } = makeMockSphinxContextForIntegrationTests([
       'contracts/test/MyContracts.sol:MyContract2',
     ])
     const { proposalRequest, parsedConfigArray, configArtifacts } =
       await propose({
         confirm: true, // Skip preview
         isTestnet,
-        isDryRun: true,
+        isDryRun: false,
```

Comment:
Changed `isDryRun` to `false` here and below because we don't allow `confirm` and `dryRun` to both be `true` anymore

Reply:
You're right that the restriction doesn't apply here. I'll just keep the check in the `proposeCommandHandler` function since neither of us are opinionated